.github/workflows/release.yaml

```diff
@@ -36,35 +36,5 @@ jobs:
       - name: Create Release Pull Request
         id: changesets
         uses: changesets/action@v1
-        with:
-          version: yarn run version
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Get Latest Tag and Publish
```

Comment:
so we're not doing releases anymore, just triggering a PR for the release ? What happens when that PR is merged?

Reply:
The changelogs for `app` and `backend` are updated when the PR is merged.

Reply:
and do changelogs trigger building and publishing new container images?

Reply:
No, only our three month release schedule/patches will trigger a new tag/publish event. The changelog release is a completely separate process.